cellpack/bin/pack.py

```diff
@@ -69,6 +86,36 @@ def pack(recipe, config_path=None, analysis_config_path=None, docker=False):
         env.buildGrid(rebuild=True)
         env.pack_grid(verbose=0, usePP=False)
 
+    if docker:
+        # copy results from original folder to unique S3 upload folder
+        if Path(env.out_folder).exists():
+            s3_upload_folder.mkdir(parents=True, exist_ok=True)
+            shutil.copytree(env.out_folder, s3_upload_folder, dirs_exist_ok=True)
+
+        upload_packing_results_to_s3(s3_upload_folder, recipe_data["name"], job_id)
+
+
+def upload_packing_results_to_s3(output_folder, recipe_name, job_id):
+    """
+    Upload packing results to S3 using the DBUploader architecture
+    :param output_folder: Path to the output folder containing results
+    :param recipe_name: Name of the recipe being packed
+    """
+    try:
+        if job_id:
+            output_path = Path(output_folder)
+            if not output_path.exists():
+                log.error(f"Output folder does not exist: {output_folder}")
+                return
+
+            uploader = DBUploader(db_handler=None)
```

Comment:
I understand that we're initializing `DBUploader` with `db_handler=None` because we need to use 2 different db_handlers in the process of upload_outputs_to_s3 (one for S3 and one for firebase), but it feels like a weird pattern to establish. Like perhaps it would be better to do the s3 uploading with an s3 db_handler and if that succeeds, initialize a new DBUploader with a firebase db_handler to update the logs?

We're on a time crunch so this probably isn't critical, but could be worth adding a TODO comment or something to come back and clean it up?